samples/tree.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Write your Calder code here
+ */
+
+const light1 = {
+    lightPosition: [10, 10, 10],
```

Comment:
I was thinking, it might be nice to use the `coord` type here instead of a three-dimensional array for consistency.

Reply:
Beyond the scope of this PR, but would be nice for the future.